azkaban-common/src/main/java/azkaban/project/ProjectLoader.java

```diff
@@ -200,4 +200,17 @@ Props fetchProjectProperty(int projectId, int projectVer, String propsName)
       throws ProjectManagerException;
 
   void updateProjectSettings(Project project) throws ProjectManagerException;
+
+  /**
+   * Uploads flow file.
+   */
+  void uploadFlowFile(int projectId, int projectVersion, int flowVersion,
```

Comment:
Can we have the same argument order as `getUploadedFlowFile`?

Reply:
They don't have exactly the same arguments, but I'll try to align them.